pom.xml

```diff
@@ -47,6 +47,7 @@
         <module>dropwizard-request-logging</module>
         <module>dropwizard-json-logging</module>
         <module>dropwizard-health</module>
+        <module>dropwizard-unix-socket</module>
```

Comment:
Is there a good reason to introduce a separate module for this change instead of integrating the class into `dropwizard-jetty`?

Reply:
mainly because the other PR did the same. If it doesn't help with anything, I can move it to jetty.

Reply:
I'm fine with that. The HTTP2 connector also resides in a separate module.